tests/utils/browser.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const feedBackDocs = async (testName = 'allLogs') => {
+  const script = `(async () => {
+  const allDbList =  await indexedDB.databases();
+  const metaDbList = allDbList.filter(db => db.name.includes('pouch_medic-user') && db.name.endsWith('-meta'))
+  const feedBack = []
+  for (const idx in metaDbList){
+    const db = metaDbList[idx]
```

Comment:
I think a `map` would be better here to avoid manually looping, eg:

```js
return allDbList
  .filter(db => db.name.includes('pouch_medic-user') && db.name.endsWith('-meta'))
  .map(db => {
    ...
    return feedBackDocs;
  })
  .flat();
```